### PR TITLE
🌐 [i18n-KO] Translated `tasks/summarization.mdx` to Korean

### PR DESCRIPTION
**Pull request by sim-so**
# What does this PR do?

Translated the `tasks/summarization.mdx` file of the documentation to Korean.
Thank you in advance for your review!

Part of #20179 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

This is a work on progress.
Could you review this PR when I finish this work?
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

### CODE REVIEW

File: docs/source/ko/tasks/summarization.mdx [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+specific language governing permissions and limitations under the License.
+-->
+
+# 요약[[summarization]]
+
+[[open-in-colab]]
+
+<Youtube id="yHnr5Dk2zCI"/>
+
+요약은 문서나 기사에서 중요한 정보를 모두 포함하여 짧은 버전으로 만드는 일입니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/summarization.mdx:19]:
```suggestion
요약은 문서나 기사에서 중요한 정보를 모두 포함하되 짧게 만드는 일입니다.
```

File: docs/source/ko/tasks/summarization.mdx [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+[[open-in-colab]]
+
+<Youtube id="yHnr5Dk2zCI"/>
+
+요약은 문서나 기사에서 중요한 정보를 모두 포함하여 짧은 버전으로 만드는 일입니다. 
+번역과 마찬가지로, 시퀀스-투-시퀀스 문제로 구성할 수 있는 대표적인 작업 중 하나입니다. 
+요약에는 아래와 같이 유형이 있습니다:
+
+- 추출(Extractive) 요약: 문서에서 가장 관련성 높은 정보를 추출합니다.
+- 생성(Abstractive) 요약: 가장 관련성 높은 정보를 포착하는 새로운 텍스트를 생성합니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/summarization.mdx:24]:
```suggestion
- 생성(Abstractive) 요약: 가장 관련성 높은 정보를 포착해내는 새로운 텍스트를 생성합니다.
```
아주 좋아요!

File: docs/source/ko/tasks/summarization.mdx [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+```
+</tf>
+</frameworkcontent>
+
+## 평가[[evaluate]]
+
+학습 중에 평가 지표를 포함하면 모델의 성능을 평가하는 데 도움이 되는 경우가 많습니다. 
+🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하면 평가 방법을 빠르게 불러올 수 있습니다. 
+이 작업에서는 [ROUGE](https://huggingface.co/spaces/evaluate-metric/rouge)을 로드합니다. 
+(평가 지표를 불러오고 계산하는 대한 자세한 내용은 🤗 Evaluate [둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요.)
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/summarization.mdx:150]:
```suggestion
(평가 지표를 불러오고 계산하는 데에 대한 자세한 내용은 🤗 Evaluate [둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요.)
```

File: docs/source/ko/tasks/summarization.mdx [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+
+<frameworkcontent>
+<pt>
+<Tip>
+
+모델을 [`Trainer`]로 파인튜닝 하는 것이 익숙하지 않다면, [여기](../training#train-with-pytorch-trainer)에서 기본 튜토리얼을 확인해보세요!
+
+</Tip>
+
+이제 모델 학습을 시작할 준비가 되었습니다! [`AutoModelForSeq2SeqLM`]로 T5를 로드하세요:
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/summarization.mdx:190]:
```suggestion
이제 모델 학습을 시작할 준비가 되었습니다! [`AutoModelForSeq2SeqLM`]로 T5를 가져오세요:
```

File: docs/source/ko/tasks/summarization.mdx [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+학습이 완료되면, 누구나 모델을 사용할 수 있도록 [`~transformers.Trainer.push_to_hub`] 메소드로 Hub에 공유합니다:
+
+```py
+>>> trainer.push_to_hub()
+```
+</pt>
+<tf>
+<Tip>
+
+Keras로 모델 파인튜닝을 하는 것이 익숙하지 않다면, [here](../training#train-a-tensorflow-model-with-keras)에서 기본적인 튜토리얼을 확인하세요!
```
Comment (wonhyeongseo) [inline docs/source/ko/tasks/summarization.mdx:244]:
```suggestion
Keras로 모델 파인튜닝을 하는 것이 익숙하지 않다면, [여기](../training#train-a-tensorflow-model-with-keras)에서 기본적인 튜토리얼을 확인하세요!
```

**Review by wonhyeongseo — CHANGES_REQUESTED** [review]
Your writing ✍️ is on-par with the best 👌  
I can read through your work with a cadence that resembles birds🕊 flying through the sky. Keep up the good work, @sim-so !

File: docs/source/ko/tasks/summarization.mdx [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+
+</Tip>
+TensorFlow에서 모델을 파인튜닝하려면, 먼저 옵티마이저, 학습률 스케줄 그리고 몇 가지 학습 하이퍼파라미터를 설정하세요:
```
[the remark this answers is not in the document]
Comment (sim-so) [reply docs/source/ko/tasks/summarization.mdx:247]: 저도 고민하다가 시간이 좀 걸릴 것 같아서 이대로 진행했습니다.
용어사전에 추가해두고, 생각나면 바꾸는 걸로 하겠습니다!

**Review by sim-so — COMMENTED** [review]
I always learn from your review! Thank you!
I changed the lines as you suggested. 😄 @wonhyeongseo

File: docs/source/ko/tasks/summarization.mdx [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+```
+</tf>
+</frameworkcontent>
+
+## 평가[[evaluate]]
+
+학습 중에 평가 지표를 포함하면 모델의 성능을 평가하는 데 도움이 되는 경우가 많습니다. 
+🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하면 평가 방법을 빠르게 불러올 수 있습니다. 
+이 작업에서는 [ROUGE](https://huggingface.co/spaces/evaluate-metric/rouge) 평가 지표를 가져옵니다. 
+(평가 지표를 불러오고 계산하는 데에 대한 자세한 내용은 🤗 Evaluate [둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요.)
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/summarization.mdx:150]: 계산하는 것에 대한
추천합니다 🙂

**Review by 0525hhgus — COMMENTED** [review]
Thank you for the good translation! 
Your translation is easy to understand 😀
I leave a short review below.

File: docs/source/ko/tasks/summarization.mdx [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+
+...     result = rouge.compute(predictions=decoded_preds, references=decoded_labels, use_stemmer=True)
+
+...     prediction_lens = [np.count_nonzero(pred != tokenizer.pad_token_id) for pred in predictions]
+...     result["gen_len"] = np.mean(prediction_lens)
+
+...     return {k: round(v, 4) for k, v in result.items()}
+```
+
+이제 `compute_metrics` 함수를 사용할 준비가 되었으며, 학습을 설정할 때 이 함수로 돌아가게 됩니다.
```
[the remark this answers is not in the document]
Comment (wonhyeongseo) [reply docs/source/ko/tasks/summarization.mdx:178]: 여기에서 `return`의 의미는 `다시 되돌아가다`에 가깝습니다. 만약 수정을 한다면,
```suggestion
이제 `compute_metrics` 함수를 사용할 준비가 되었으며, 학습을 설정할 때 이 함수로 되돌아올 것입니다.
```

[the remark this answers is not in the document]
Comment (sim-so) [reply docs/source/ko/tasks/summarization.mdx:178]: 두 분 모두 감사합니다!
원형님이 제안해주신 내용으로 반영하겠습니다!

File: docs/source/ko/tasks/summarization.mdx (hunk 7 above)
Comment (sim-so) [reply docs/source/ko/tasks/summarization.mdx:150]:
```suggestion
(평가 지표를 불러오고 계산하는 방법은 🤗 Evaluate [둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하세요.)
```
제안 감사합니다!
계속 읽어보니 문장을 간결하게 하고 원문에 있던 `how to`를 살리면 좋을 것 같아 이렇게 수정하려고 해요.

**Comment by sim-so** [comment]
Could you review this PR? 😄 
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!
